### PR TITLE
Rebrand stable9

### LINKED ISSUE #426: Rebrand stable9 to Nextcloud
- [ ] Rebrand README.md 
- [ ] Rebrand issue_template.md

@MorrisJobke Do we have badges (build, ...) for `stable9`?

### PR DESCRIPTION
**Pull request by MorrisJobke**
- adjust issue_template and README
- fixes #426 

cc @LukasReschke @nickvergessen @mar1u5

### CODE REVIEW

**Comment by nickvergessen** [comment]
:+1:

**Comment by LukasReschke** [comment]
👍

File: issue_template.md [hunk 1]
```diff
@@ -25,40 +27,49 @@ Tell us what happens instead
...
-**ownCloud version:** (see ownCloud admin page)
+**Nextcloud version:** (see Nextcloud admin page)
 
-**Updated from an older ownCloud or fresh install:**
+**Updated from an older Nextcloud/ownCloud or fresh install:**
 
-**Where did you install ownCloud from:**
+**Where did you install Nextcloud from:**
 
-**Signing status (ownCloud 9.0 and above):**
```
Comment (mar1u5) [inline issue_template.md]: There is nothing under 9 :stuck_out_tongue_winking_eye: See #425

Comment (MorrisJobke) [inline issue_template.md]: That is, why this is removed :P

Comment (mar1u5) [inline issue_template.md]: :unamused: ... I should definetly not comment while i am on my mobile :stuck_out_tongue_winking_eye:

**Pull request merged**